Game/Resources/Shaders/simple_color.fs

```diff
@@ -0,0 +1,8 @@
+uniform vec4 base_color;
+
+out vec4 color;
+
+void main()
+{
+    color = base_color;
```

Comment:
What is this fs for?

Reply:
This is used for Outline efect in editor and also in the NavMesh debug visualization